### PR TITLE
Package files for S.S.C.ProtectedData

### PR DESCRIPTION
**Pull request by AtsushiKan**

### CODE REVIEW

**Comment by AtsushiKan** [comment]
cc @ericstj

**Comment by AtsushiKan** [comment]
cc @bartonjs

File: src/System.Security.Cryptography.ProtectedData/pkg/System.Security.Cryptography.ProtectedData.pkgproj [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <ProjectReference Include="..\ref\System.Security.Cryptography.ProtectedData.csproj">
+      <SupportedFramework>net46;netcore50;netstandardapp1.5</SupportedFramework>
+    </ProjectReference>
+    <ProjectReference Include="..\src\System.Security.Cryptography.ProtectedData.builds" />
+    
+    <ExcludeDefaultValidateFramework Include="netstandardapp1.5" />
+    <!-- Only supported on windows, so overriding
+         the validation RID's -->
+    <ValidateFramework Include="netstandardapp1.5">
+      <RuntimeIDs>win7-x86;win7-x64</RuntimeIDs>
```
Comment (bartonjs) [inline src/System.Security.Cryptography.ProtectedData/pkg/System.Security.Cryptography.ProtectedData.pkgproj]: @ericstj Given the new auto-PNSE libraries (with your oh-so-colorful metaphor of a name :)) is this still the way to do this?

File: src/System.Security.Cryptography.ProtectedData/pkg/System.Security.Cryptography.ProtectedData.pkgproj [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Security.Cryptography.ProtectedData.csproj">
+      <SupportedFramework>net46;netcore50;netstandardapp1.5</SupportedFramework>
```
Comment (bartonjs) [inline src/System.Security.Cryptography.ProtectedData/pkg/System.Security.Cryptography.ProtectedData.pkgproj]: @ericstj If the library is netstandard1.3, shouldn't it be netstandardapp1.3 here?

**Comment by bartonjs** [comment]
There's nothing that I know needs to change, but I don't have Eric's expertise to say if it's "correct".  So please let Eric address my questions before merging :)

File: src/System.Security.Cryptography.ProtectedData/pkg/System.Security.Cryptography.ProtectedData.pkgproj (hunk 2 above)
Comment (ericstj) [inline src/System.Security.Cryptography.ProtectedData/pkg/System.Security.Cryptography.ProtectedData.pkgproj]: No, SupportedFrameworks element is just an key/TFM into this list of TFM/RID tuples that we use for testing support from the package. https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets#L756

File: src/System.Security.Cryptography.ProtectedData/pkg/System.Security.Cryptography.ProtectedData.pkgproj (hunk 1 above)
Comment (ericstj) [inline src/System.Security.Cryptography.ProtectedData/pkg/System.Security.Cryptography.ProtectedData.pkgproj]: We should build a notsupported assembly for this guy.  I can help with that.

Comment (ericstj) [inline src/System.Security.Cryptography.ProtectedData/pkg/System.Security.Cryptography.ProtectedData.pkgproj]: Here's a sample: https://github.com/ericstj/corefx/tree/AtsushiKan-pkg

We can do it as a seperate PR if you like.

Comment (AtsushiKan) [inline src/System.Security.Cryptography.ProtectedData/pkg/System.Security.Cryptography.ProtectedData.pkgproj]: Yes, let’s do it as a separate PR

Comment (ericstj) [inline src/System.Security.Cryptography.ProtectedData/pkg/System.Security.Cryptography.ProtectedData.pkgproj]: Ok

**Comment by ericstj** [comment]
Lgtm
